docs/en/guide/README.md

```diff
@@ -0,0 +1,15 @@
+# Guide
+
+If you want to debug, develop or execute SkyWalking Infra E2E, The following documentation would guide you.
+
+## Compile
+
+* [How to build?](compile/How-To-Build.md)
+
+## Execute
+
+The most important thing in E2E Testing is that it uses a separate configuration file and command to execute.
+If you haven't read the [Module Design](../concepts-and-designs/module-design.md), recommend read this document first.
+
+* [How to write configuration file?](execute/How-To-Write-Configuration-File.md)
+* [How to execute?](execute/How-To-Execute.md)
```

Comment:
Please update all `how to` sentences and file names into statements, such as `How-To-Write-Configuration-File.md` to `configuration-file.md`. Because `how-to` prefix is usually used in FAQ or blog.